doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
  */
 ////
 
-calcurse(1)
+CALCURSE(1)
 ===========
 
-Name
+NAME
 ----
 
-calcurse - text-based organizer
+calcurse - terminal-based organizer for interactive and command line use
 
-Synopsis
+SYNOPSIS
 --------
 
-[verse]
-'calcurse' *-Q* [options] [*--from* <date>] [*--to* <date>|*--days* <num>]
-'calcurse' *-G* [options]
-'calcurse' *-i*<file>
-'calcurse' *-x*<format>
-'calcurse' *--gc*
-'calcurse' *--status*
-'calcurse' *--version*
-'calcurse' *--help*
-
-Description
------------
+--
+*calcurse* [-D 'directory'] | [-C 'confdir'] | [-c 'calendar-file']
+
+*calcurse* -Q [*--from* 'date'] [*--to* 'date' | *--days* 'number']
+
+*calcurse* -a | -d 'date' | -d 'number' | -n | -r['number'] | -s['date'] | -t['number']
+
+*calcurse* -h | --status | -G | -g | -i 'file' | -P | -x['file'] | --daemon
+--
 
-Calcurse is a text-based calendar and scheduling application. It helps keeping
-track of events, appointments and everyday tasks.  A configurable notification
-system reminds user of upcoming deadlines, and the curses based interface can
-be customized to suit user needs.  All of the commands are documented within an
-online help system.  
+Briefly, the first form shows how to invoke calcurse interactively; the
+remaining are command line forms. The second form shows queries, the third
+short-form queries; both may be combined with filter options as well as
+format options. The fourth form is miscellaneous help and support functions.
+All details are in <<_options,OPTIONS>>.
 
-Options
+DESCRIPTION
+-----------
+
+Calcurse is a calendar and scheduling application for use in a terminal
+session (terminal emulator). When invoked without options, calcurse enters
+interactive mode; in most other cases, calcurse produces output on the
+terminal and exits.  It helps keeping track of events, appointments and
+everyday tasks. Interactive mode is used when data are entered or when already
+existing entries are inspected or edited. All data are saved to disk as text
+files. Command line mode is used for queries and administrative tasks and for
+automating tasks in scripts.
+
+The interactive interface is based on ncurses and can be customized to suit
+user preferences. Customization includes program behaviour as well as visual
+appearance and key bindings, and is performed interactively; the result is
+automatically saved to disk and loaded on every invocation. Available actions
+are documented in an online help system. A configurable notification system
+issues reminders of upcoming deadlines.
+
+When leaving the interactive program, a background daemon may continue running
+and issue notifications; it stops automatically when the interactive mode is
+reentered.
+
+This man-page mainly describes the command-line mode.
+
+OPTIONS
 -------
 
-The following options are supported:
+.Input and Output Date Format
+*****************************
```

Comment:
I'd skip that heading. It may look nice and consistent in the generated HTML but the primary purpose of this document is to serve as a man page and the additional heading is very uncommon there.

Reply:
I admit that I mainly tried to make the HTML version look nice at the expense of the traditional terminal version. I have long since lost contact with the day-to-day IT world. Do (Unix) people still read man pages with the man command? Anyway, I have gone back to a traditional layout with just a few section levels (three).

Reply:
Yes, I believe that people still rely on man pages, especially when looking for a specific command line option. Of course, Google often helps as well but I'd say man pages are still being used, especially among heavy shell users (which is one of the main target groups of calcurse).

Beyond that, this is the source code for the man page. If we decide that we want to drop the man page and replace it with HTML documentation, we should do that instead of keeping the man page and making it ugly, while trying to make it convert into a pretty HTML page .